docs/configuration/registration/camunda.rst

```diff
@@ -0,0 +1,73 @@
+.. _configuration_registration_camunda:
+
+=======
+Camunda
+=======
+
+Camunda_ is a process engine using BPM models. Open Forms supports Camunda 7.x
+
+.. note::
+
+    This service requires a connection to an external Camunda instance, offered
+    or maintained by a service provider.
+
+What does the Open Forms administator need?
+===========================================
+
+The values for these parameters should be provided to the Open Forms
+administrator by the service provider.
+
+============================  =======================================================================================
+Parameter                     Description
+============================  =======================================================================================
+**Camunda instance**
+Version                       7.x. 7.16 is actively tested, 7.12 and up are expected to work. Other 7.x versions *should* work.
+REST API root endpoint        The Camunda instance must expose its REST API on a URL reachable by Open Forms.
+Username                      Username for the REST API user (Basic Auth)
+Password                      Password for the REST API user (Basic Auth)
```

Comment:
This should match the configuration that is implemented. My suggestion is not table ready!
The version is not part of this, but is simpy a requirement from the plugin.

```suggestion
Camunda root        The domain where the Camunda instance runs.
REST api path           The path where the API of the above Camunda instance can be found.
Username                      Username for the REST API user (Basic Auth)
Password                      Password for the REST API user (Basic Auth)
```